store/rootmulti/store.go

```diff
@@ -275,8 +273,7 @@ func (rs *Store) loadVersion(ver int64, upgrades *types.StoreUpgrades) error {
 			// handle renames specially
 			// make an unregistered key to satisfy loadCommitStore params
 			oldKey := types.NewKVStoreKey(oldName)
-			oldParams := storeParams
-			oldParams.key = oldKey
+			oldParams := NewStoreParams(oldKey, storeParams.db, storeParams.typ, 0)
```

Comment:
Maybe it is a stupid question, but why is 0 and not `storeParams.initialVersion-1` for the initial version?
It has been bumped at L254.

Reply:
Great Q @julienrbrt. @catShaark is this because the store already exists and is just being renamed? Let's be sure to have a test that tests this edge case.

Reply:
This is because we're loading the old store so that we can move data from that old store to the new store. If we want to load an already exist store, we must set initial version to 0 or else we'll have `initial version set to %v, but found earlier version %v` error

Reply:
there's already a test for store rename which is `TestSetLoader` in x/upgrades/types

Reply:
> there's already a test for store rename which is TestSetLoader in x/upgrades/types

Right, but haven't you remove the case for the store rename? https://github.com/cosmos/cosmos-sdk/pull/14410/files#diff-d7f6a3ea9905545b1e8afd31dcd8f9fccf72124c43349d647c3bae894d4d6114L151

Reply:
So there're 2 checks, the first one checks if we can load the root store with the old store mounted, the second checks if we can load the root store with the new store mounted. I only remove the first one and also we only need the second one to confirm that new store is of correct height.